test/indices.jl

```diff
@@ -202,4 +202,12 @@
   @test linds[2, 2] == only(indices(grid, p4))
   @test linds[10, 10] == only(indices(grid, p5))
   @test isempty(indices(grid, p6))
+
+  # fast method for transformed grids with invertible transforms
+  tri = Triangle(cart(5, 5), cart(10, 10), cart(15, 5))
+  grid = cartgrid(20, 20)
+  trans = Translate(2.0, 3.0)
+  ttri = TransformedGeometry(tri, trans)
+  tgrid = TransformedGrid(grid, trans)
+  @test issetequal(indices(tgrid, transtri), indices(grid, tri))
```

Comment:
Can we be more strict and use `==` in this case? My intuition says that the translation shouldn't change the order of indices.